modules/update-cluster-version-object.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="update-cluster-version-object_{context}"]
+= The ClusterVersion object
+
+One of the resources that the Cluster Version Operator (CVO) monitors is `ClusterVersion`.
```

Comment:
nit: the CVO monitors many resources, but I'd break them into a few classes:

* Resources that the CVO manages, where we are opinionated about some `metadata` and `spec` properties. 
 Some other controller attempts to continuously deliver the CVO's requests and reports the results in `status`.  The CVO consumes `status` and bubbles up any issues in ClusterVersion's `status`.
 * Resources that configure the CVO's operation.  Mostly this is ClusterVersion, where folks who want to manage the CVO set `spec`, and the CVO attempts to continuously deliver those requests and reports the results in `status`.  But the CVO also consumes configuration from other resources, e.g. Proxy, to figure out how the cluster admins want us to reach off-cluster for new update recommendations and similar.
* A small number of resources are both managed by and configuring the CVO, e.g. the `admin-gates` ConfigMap, where we set some keys we care about, but allow the cluster admin to inject additional keys, and we consume and respect those additions too.

Can we start hinting at this with:

> The ClusterVersion resource is the primary interface for managing the Cluster Version Operator (CVO).  Cluster administrators and other controllers may declare their desired state through ClusterVersion `spec` and hear about how the CVO is delivering those requests in `status`.

or something else that says "this is where you steer the CVO" without dropping into "and the CVO in turn will go manage some other resources for you" sorts of things?  We can get at those other resources later, but I think pulling them in now seems overly complicated.

Reply:
I'll replace the first sentence in the following paragraph with this (and maybe still find a way to mention that it contains version info)